socialhome/users/models.py

```diff
@@ -272,15 +272,25 @@ def get_last_name(self):
         return ""
 
     @staticmethod
+    def absolute_remote_url(remote_profile, image_name):
```

Comment:
Could the signature be `absolute_image_url(profile, image_name)`? Reason: while we're using this for remote profile parsing, the functionality in the method doesn't actually have anything strictly to do with remote profiles. But it does have everything to do with image url's, so calling it `absolute_image_url` would make it easier to see right from the name what it does. With the current name my first thought is that this returns "an absolute url for the remote profile itself".

Also, a short docstring would be nice, just to indicate what it does (= returns absolute version of image ulr of given size).